spec/services/wallets/balance/increase_service_spec.rb

```diff
@@ -5,7 +5,15 @@
 RSpec.describe Wallets::Balance::IncreaseService, type: :service do
   subject(:create_service) { described_class.new(wallet:, credits_amount:) }
 
-  let(:wallet) { create(:wallet, balance_cents: 1000, credits_balance: 10.0) }
+  let(:wallet) {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://rubystyle.guide#single-line-blocks)